Doc/reference/import.rst

```diff
@@ -922,6 +922,53 @@ it is sufficient to raise :exc:`ModuleNotFoundError` directly from
 while raising an exception terminates it immediately.
 
 
+Package Relative Imports
+========================
+
+Relative imports use leading dots. A single leading dot indicates a relative
+import, starting with the current package. Two or more leading dots indicate a
+relative import to the parent(s) of the current package, one level per dot
+after the first. For example, given the following package layout::
+
+    package/
+        __init__.py
+        subpackage1/
+            __init__.py
+            moduleX.py
+            moduleY.py
+        subpackage2/
+            __init__.py
+            moduleZ.py
+        moduleA.py
+
+Given that the current file is either ``moduleX.py`` or ``subpackage1/__init__.py``,
+the following is the syntax for relative imports:
+
+    from .moduleY import spam
+    from .moduleY import spam as ham
+    from . import moduleY
+    from ..subpackage1 import moduleY
+    from ..subpackage2.moduleZ import eggs
+    from ..moduleA import foo
+    from ...package import bar
+
+Relative imports must always use ``from <> import``; ``import <>`` is always absolute.
```

Comment:
I think (and I emphasize *think* here, because I'm not convinced this is right) this sentence and the next one can be reworked a bit to flow better by starting with the behavior of absolute import, then going to relative imports, so something like this:

"Absolute imports may use either the `import <>` or `from <> import <>` syntax, but relative imports may only use the second form; the reason for this is that 

```python
    import XXX.YYY.ZZZ
```

should expose `XXX.YYY.ZZZ` as a usable expression, but `.moduleY` is not a valid expression."

Hard to tell if I like it better without looking at it with fresh eyes later, but I thought I'd throw it out there as an option in case you do.